### PR TITLE
Migrate CockroachDBConfigStore to @configu/integrations 

### PR DESCRIPTION
**Pull request by gagandeepp**
Closes https://github.com/configu/configu/issues/513

### CODE REVIEW

**Comment by gagandeepp** [comment]
@rannn505  Please review

File: packages/integrations/stores/cockroach-db/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  },
+  "homepage": "https://configu.com",
+  "keywords": [
+    "cfgu",
+    "configu"
+  ],
+  "scripts": {
+    "build": "tsup"
+  },
+  "dependencies": {
```
Comment (RichardAkman) [inline packages/integrations/stores/cockroach-db/package.json:24]: `typeorm` is missing in the dependencies.

File: packages/integrations/stores/cockroach-db/src/cockroach-db.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { type CockroachConnectionOptions } from 'typeorm/driver/cockroachdb/CockroachConnectionOptions';
+import { ORMConfigStore, type ORMConfigStoreSharedConfiguration } from './ORM';
```
Comment (RichardAkman) [inline packages/integrations/stores/cockroach-db/src/cockroach-db.ts:2]: Please align these with other integrations that base off of `typeorm`:
```suggestion
import { type CockroachConnectionOptions } from 'typeorm/driver/cockroachdb/CockroachConnectionOptions.js';
import { ORMConfigStore, type ORMConfigStoreSharedConfiguration } from '@configu/integrations/src/utils/ORM';
```

**Review by RichardAkman — CHANGES_REQUESTED** [review]
Great work ⭐ 

Please review my minor comments.

File: packages/integrations/stores/cockroach-db/package.json (hunk 1 above)
Comment (gagandeepp) [reply packages/integrations/stores/cockroach-db/package.json:24]: Done

File: packages/integrations/stores/cockroach-db/src/cockroach-db.ts (hunk 2 above)
Comment (gagandeepp) [reply packages/integrations/stores/cockroach-db/src/cockroach-db.ts]: Done

**Comment by gagandeepp** [comment]
@rannn505 please review

**Comment by gagandeepp** [comment]
@davidmeirlevy  please review

**Comment by RichardAkman** [comment]
@gagandeepp, seems like you resolved everything but one thing is missing. Please run `pnpm i` on the repository root folder to ensure the `pnpm-lock.yaml` is updated, this is a requirement to pass the CI.

**Review by RichardAkman — APPROVED** [review]
LGTM, I'll handle the update of the lock file.

**Pull request merged**